builder/job.go

```diff
@@ -44,6 +44,7 @@ var validCommitCommands = map[string]bool{
 	"workdir":    true,
 }
 
+// Config containers all configs for a build job
```

Comment:
LOL this happens to me all the time - my fingers get so used to typing a certain word that when I try to type a slight variant of it I have to think hard to get it right :-)